known-and-unknown.md

```diff
@@ -123,255 +101,246 @@ following the definition of VCS uri from the
-
-We can run the query:
-
-**Note**: `source` is specified to indicate we are querying a source repo
-
-```bash
-./bin/guacone query known source "git+https://github.com/prometheus/client_golang"
-```
-
-```bash
-+-----------+-----------+--------------------------------------------------------------------+
-| NODE TYPE | NODE ID   | ADDITIONAL INFORMATION                                             |
-+-----------+-----------+--------------------------------------------------------------------+
-| hasSrcAt  | 7647      | Source for Package: pkg:golang/github.com/prometheus/client_golang |
-+-----------+-----------+--------------------------------------------------------------------+
-| scorecard | 7583      | Overall Score: 6.600000                                            |
-+-----------+-----------+--------------------------------------------------------------------+
-Visualizer url: http://localhost:3000/?path=7582,7581,6965,7647,7583
-```
-
-From this output, we see the reverse of what we saw when we queried the package.
-This time, we see that this source is related to the prometheus/client_golang
-package we queried for above but also we see that there is an OpenSSF scorecard
-associated.
-
-Finally, let’s query for another source repo:
-
-```bash
-./bin/guacone query known source "git+https://github.com/googleapis/google-cloud-go"
-```
-
-The output should be similar to:
-
-```bash
-+-----------+-----------+---------------------------------------------------------------------+
-| NODE TYPE | NODE ID   | ADDITIONAL INFORMATION                                              |
-+-----------+-----------+---------------------------------------------------------------------+
-| hasSrcAt  | 7074      | Source for Package: pkg:golang/cloud.google.com/go                  |
-| hasSrcAt  | 7075      | Source for Package: pkg:golang/cloud.google.com/go/storage          |
-| hasSrcAt  | 7156      | Source for Package: pkg:golang/cloud.google.com/go/spanner          |
-| hasSrcAt  | 8948      | Source for Package: pkg:golang/cloud.google.com/go/compute/metadata |
-| hasSrcAt  | 8949      | Source for Package: pkg:golang/cloud.google.com/go/logging          |
-| hasSrcAt  | 8950      | Source for Package: pkg:golang/cloud.google.com/go/longrunning      |
-+-----------+-----------+---------------------------------------------------------------------+
-| scorecard | 6968      | Overall Score: 8.300000                                             |
-+-----------+-----------+---------------------------------------------------------------------+
-Visualizer url: http://localhost:3000/?path=6967,6966,6965,7074,7075,7156,8948,8949,8950,6968
-```
-
-Here we see that this specific source repo is associated with various different
-packages (go, storage, spanner…etc), and again we see that it does have a
-scorecard associated with a score of 8.3
+1. Look at if a package (vault) to see if it has an SBOM associated with and where it can be found:
```

Comment:
Link to the SBOM we are ingesting for Vault for reference? ((https://github.com/guacsec/guac-data/blob/main/top-dh-sboms/vault.json))